corgi/tasks/pnc.py

```diff
@@ -30,4 +36,82 @@ def slow_fetch_pnc_sbom(purl: str, product_data, build_data, sbom_data) -> None:
     response.raise_for_status()
     logger.info(f"SBOM fetched for purl {purl}")
 
-    parse_pnc_sbom(response.json())
+    # Make sure the SBOM is valid
+    sbom = SbomerSbom(response.json()["sbom"])
```

Comment:
Minor nit: let's break this onto two lines so it's easier to see which one fails in a traceback:
```suggestion
    # Make sure the SBOM is valid
    sbom_json = response.json()["sbom"]
    sbom = SbomerSbom(sbom_json)
```